openlibrary/plugins/upstream/addbook.py

```diff
@@ -605,21 +631,25 @@ def _prevent_ocaid_deletion(self, edition):
             logger.warn("Attempt to change ocaid of %s from %r to %r.", self.edition.key, self.edition.get('ocaid'), ocaid)
             raise ValidationException("Changing Internet Archive ID is not allowed.")
 
-    def do_work_keys_match(self, i):
+    @staticmethod
+    def is_work_data_important(formdata):
         """
         Check if the form data's work matches the form data's edition's work.
-        :param web.storage i: form data
+        If they're not, then we ignore the work edits.
```

Comment:
How is the user informed that their edits are being ignored?

Reply:
![image](https://user-images.githubusercontent.com/6251786/57718853-65431d00-764c-11e9-8e2c-e5062479c395.png)